### DISCUSSION TITLE
Exclude specific page from search results

### DISCUSSION THREAD

**Discussion by sewid** [statamic/cms#6194] [Q&A] [answered]
Hi there,

is it possible to exclude one or two specific pages from search results when all other pages should be found? I could not find any information about this topic in the docs.

Thanks and best regards
Sebastian

**Comment by jasonvarga** [comment, marked as answer]
Sure, use a condition.

`{{ search:results id:not_in="id_one|id_two" }}`

https://statamic.dev/conditions

**Reply by sewid** [reply]
It's so simple :-) Thanks a lot!
